src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -676,3 +677,13 @@ void CoilCoolingDXCurveFitPerformance::setOperMode(EnergyPlus::EnergyPlusData &s
                               this->object_name));
     }
 }
+void CoilCoolingDXCurveFitPerformance::oneTimeAvailSchedSetup(EnergyPlus::EnergyPlusData &state)
+{
+    if (this->myOneTimeAvailSchedInitFlag) {
+        // set avail schedule for each mode
+        this->normalMode.coilCoolingDXAvailSched = static_cast<EnergyPlus::CoilCoolingDXPerformanceBase *>(this)->coilCoolingDXAvailSched;
+        this->alternateMode.coilCoolingDXAvailSched = this->normalMode.coilCoolingDXAvailSched;
+        this->alternateMode2.coilCoolingDXAvailSched = this->normalMode.coilCoolingDXAvailSched;
+        this->myOneTimeAvailSchedInitFlag = false;
+    }
+}
```

Comment:
This new function propagates the availability schedule pointer for each operating mode.